stages/test/test_containers_storage_conf.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/python3
+
+import os.path
+import pytoml
```

Comment:
I think the linter wants pytoml next to pytest not os.path since it's a 3rd party library. Too bad it doesn't actually explain itself :)